docs/03-tutorials/00-application-connectivity/ac-05-call-registered-service-from-kyma.md

```diff
@@ -37,12 +39,11 @@ This guide shows how to call a registered external service from Kyma using a sim
    {"id":"{YOUR_SERVICE_ID}"}
    ```
 
-2. Export the ID of the registered service, the Namespace to which you bound your Application, and the name of your Application.
+2. Export the ID of the registered service and the Namespace in which to create the Function that calls the service as environment variables:
    
    ```bash
    export SERVICE_ID={YOUR_SERVICE_ID}
    export NAMESPACE={YOUR_NAMESPACE}
```

Comment:
I added exporting the Namespace to an env var back here since I mistakenly remove it before.